ci-operator/step-registry/upi/deprovision/heterogeneous/powervs/upi-deprovision-heterogeneous-powervs-commands.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+
+# var.tfvars used to provision the powervs nodes is copied to the ${SHARED_DIR}
+echo "Invoking upi deprovision heterogeneous powervs"
+
+IBMCLOUD_HOME_FOLDER=/tmp/ibmcloud
+mkdir -p ${IBMCLOUD_HOME_FOLDER}
+REGION="${LEASED_RESOURCE}"
+
+if [ -z "$(command -v ibmcloud)" ]; then
+  echo "ibmcloud CLI doesn't exist, installing"
+  curl -fsSL https://clis.cloud.ibm.com/install/linux | sh
+fi
+
+function ic() {
+  HOME=${IBMCLOUD_HOME_FOLDER} ibmcloud "$@"
+}
+
+ic version
+ic login --apikey @${CLUSTER_PROFILE_DIR}/ibmcloud-api-key -r ${REGION}
+ic plugin install -f cloud-internet-services vpc-infrastructure cloud-object-storage power-iaas is
+
+echo "Check if the var.tfvars exists in ${SHARED_DIR}"
+if [ -f "${SHARED_DIR}/var.tfvars" ]
+then
+  # upgrade go to 1.20.7 or greater
```

Comment:
A refactor of this would likely see us pull out these versions so that we can update them in one of the yaml files that passes vars to this shell file. This way as openshift versions are bumped, these are easy to maintain.